data/phi0631/phi001/phi0631.phi001.perseus-eng2.xml

```diff
@@ -29,13 +29,14 @@
 <monogr>
 <title>Conspiracy of Catiline</title>
 <author>Sallust</author>
-<editor role="transl">Rev. John Selby Watson, M.A.</editor>
+<editor>Rev. John Selby Watson, M.A.</editor>
```

Comment:
Isn't he the translator? or just the editor? Translations should typically have a translator

Reply:
Strange, I thought I had fixed that. Will fix now.